### ISSUE TITLE
User can see an enabled/disabled of an entire Screen (if my toggle feature level allow)

### ISSUE THREAD

**Issue by corinnekrych** [openshiftio/openshift.io#1667]
Belong to [EPIC: Feature Flags](https://github.com/openshiftio/openshift.io/issues/1165)
# Description
Partly done in spike.
As a user with internal, I should see the screen with the 'internal' banner.
As a user with experimental, I should see the screen with the 'experimental' banner (green).
As a user with beta, I should see the screen with the 'beta' banner.
# Acceptance Criteria
List of tasks:
- [x] implement ccs for color for internal(red), beta(green), experimental (yellow)
- [x] implement different toggle level in f8-ui POC branch
- [x] implement different toggle banner in f8-ui
- [x] make a decision (and implement correctly) whether we want to do an auth call or use JWT token
- [x] proxy endpoint in `wit` for `api/feature` endpoint (#1796)
- [x] refactor f8-toggles-service to use auth-service to check groupid / retrieve user's profile from auth (#1716)
- [x] manage the feature group a user belongs to (#1717)
- [x] expose toggled feature information in the service API (#1730)
- [x] hide menu for features that are toggled off by admin
- [x] hide features-flagged menu for non-logged-in users
- [x] add support for `released` level of feature (#1912)

cc @alexeykazakov @aslakknutsen @xcoulon

**Comment by corinnekrych** [comment]
Wether we want to do an auth call or use JWT token was capture with this dagram:
![33674135-f42e4a34-daae-11e7-8c97-8279da421445](https://user-images.githubusercontent.com/1395710/33988492-c6fff834-e0c4-11e7-8826-8730f99e0dc3.png)

In the POC we use JWT token with hard coded JWT token. whereas the diagram show http REST endpoint.
Let's make a decision on that.
cc @alexeykazakov @aslakknutsen @xcoulon

**Comment by corinnekrych** [comment]
After discussing wether we can use JWT token (with a new embedded custom claim) vs we need to use auth service, we discussed the following arguments:
* pro for JWT
 * decoupling of services, potentially toggles could be used in other backend services
 * not adding any dependencies to f8-toggles-service with auth-service. 
* con for JWT:
 * UI will need to handle the refresh of token on user consent flow (should be easy to implement with refresh endpoint)
 * more importantly, auth-service will need to generate a new custom claim. Atm, auth-service is not generating its own JWT token (heavily rely on keycloak token), refactor to use own JWT token will be needed prior to implement new custom claims support.

As a conclusion, we (@alexeykazakov @aslakknutsen @xcoulon and @corinnekrych) think:
* for first iteration of f8-toggle-service, we need to revisit the service to not use JWT token but does an extra http call to `/api/user` endpoint via auth-service (as suggestion by @xcoulon potentially cache this call to limit http calls). f8-toggle-service will add the extra dependency. auth-service will provide an extra column in user attribute and add it to the /api/user payload. @alexeykazakov do you have a link to a separate issue to track this?
* middle term, Auth being a crosscutting concern, all services need it, therefore the usage of JWT token (embedding custom claims) is a nice way to achieve decoupling. Once auth-service is ready we'll drop auth-dependency and work with JWT token with a groupId custom claim.
